### ISSUE TITLE
Game for the game showcase

### ISSUE THREAD

**Issue by HelperWesley** [4ian/GDevelop-website#136]
Hello there.

I made a free game with GDevelop for a game jam, and it was very well received within the VimJam community on itch.io.
It's currently sitting as the 7th "most rated" game of itch.io's 14th largest game jam ever.

![ezgif com-optimize (1)](https://user-images.githubusercontent.com/72082749/94560686-915d3f00-023d-11eb-9b05-7029b386639d.gif)
![InGame1](https://user-images.githubusercontent.com/72082749/94560693-928e6c00-023d-11eb-9e17-5c044f1d3d68.png)
![InGame2](https://user-images.githubusercontent.com/72082749/94560698-93270280-023d-11eb-9ae1-30ee563f8f5f.png)
![InGame3](https://user-images.githubusercontent.com/72082749/94560699-93270280-023d-11eb-93de-5d7d0939aa44.png)

My online name is HelperWesley, you've probably seen me hanging out and helping people in the discord.

My game's name is "In And Out-Night Burglar", and I think it would be a great example of what can be done in just **_7 days_** with Gdevelop's quick and easy to use interface.

[https://helperwesley.itch.io/inandout-night-burglar](url)

**Comment by HelperWesley** [comment]
The game placed 20th in the "fun score" and 41st overall out of 470 games.

I call that a decent submission. ;)

**Comment by 4ian** [comment]
Congrats! Great job! 👏

I've added this to my todo list, though feel free to open a PR to add it to the showcase. We could make it as one of the featured games in the top of the page :)

**Comment by HelperWesley** [comment]
Thank you!

I have to go to work now, but when I get back I'll look in to that. I'd be honored to have one of my games featured.

By PR you mean pull request right? Just to clarify. lol

**Comment by 4ian** [comment]
Yes pull request :) If you never done one, take a look at the README first: https://github.com/4ian/GDevelop-website and try to see if you can install the development tools to make changes locally (your game should be added here: https://github.com/4ian/GDevelop-website/blob/master/src/pages/games-showcase.js). Then you can open a Pull Request to submit your changes (see https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).

**Comment by 4ian** [comment]
If you struggle too much I'll do it :)
